packages/oft-evm/README.md

```diff
@@ -0,0 +1 @@
+# OFT EVM
```

Comment:
Could you copy the standard README from one of the other packages? Consistency is key :D